src/css/controls/imports/jwplayerlayout.less

```diff
@@ -17,7 +17,7 @@
     transition: opacity 250ms cubic-bezier(0, 0.25, 0.25, 1);
     z-index: 0;
 
-    .jwplayer.jw-state-playing.jw-flag-user-inactive:not(.jw-flag-casting) & {
+    .jwplayer.jw-state-playing.jw-flag-user-inactive:not(.jw-flag-casting):not(.jw-flag-ads) & {
```

Comment:
We should look into reducing the specificity by removing `.jwplayer`. 

Also, though the skip button should be visible, the other controls aren't visible during ad playback when `.jw-flag-user-inactive` is present.